presto-docs/src/main/sphinx/release/release-347.md

```diff
@@ -0,0 +1,55 @@
+# Release 347 (XX Nov 2020)
+
+## General Changes
+
+* Add `ALTER VIEW ... SET AUTHORIZATION` syntax to allow changing the view owner. ({issue}`5789`)
+* Add support for `ALL` qualifier for `INTERSECT` and `EXCEPT`. ({issue}`2152`)
+* Add {func}`contains_sequence` function. ({issue}`5593`)
+* Add support for configuring subnet based network topology for topology-aware scheduling. ({issue}`4862`) 
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`5905`, {issue}`5949`)
+* Improve result throughput over fast network links by making http response compression configurable. Clients
```

Comment:
Maybe rephase:
```
Allow to disable client HTTP results compression, which can improve throughput over fast network links.
Compression can be disabled in CLI using the `--disable-compression` flag.
In JDBC driver it can be disabled using the `disableCompression` driver property.
```

Reply:
I rephrased it a bit based on your suggestion.